frontend/src/components/Workspace/Visualize/VisualizeItem.tsx

```diff
@@ -80,7 +91,7 @@ export const VisualizeItem = memo(function VisualizeItem({
               isSelected ? theme.palette.primary.light : undefined,
           }}
         >
-          <ItemHeader itemId={itemId} />
+          <ItemHeader itemId={itemId} setRoiFilePath={setRoiFilePath} />
           <DisplayDataItem itemId={itemId} />
```

Comment:
@sanglevinh 
Is the cancel API currently implemented according to the following?
-> It seems that the cancel API is also called when switching items with "Select Item", so please let me first check the current implementation status.

https://github.com/arayabrain/barebone-studio/pull/106/files/c89898796c08bf766fc341fc350915b4edb21133#r1414881532
> - press Cancel,
> - Press the button Close (X)
> - switch tab (e.g. Visualize -> Record)

A reproduced image is attached.
- https://github.com/arayabrain/barebone-studio/assets/122338895/5ff2cef9-0036-4bca-a814-9f04aa8ba06d

- Environment to check if it works
  - Target branch: merge latest version of feature/add_ROI_edit and improve-edit-roi
  - Environment: Mac, Ubuntu

Reply:
@itutu-tienday 
yes in this case the unmount is called by unmount()
Sorry, I missed it in the list of cases where cancel() was run